creation/web_base/condor_vars.lst

```diff
@@ -70,6 +70,8 @@ GLIDECLIENT_Rank	C	1		+				N	N	-
 GLIDECLIENT_Group_Rank	C	1		+				N	N	-
 GLIDEIN_CLAIM_WORKLIFE_DYNAMIC	I	3600		+				N	N	-
 GLIDEIN_CLAIM_WORKLIFE	I	-1		+				N	N	-
+GLIDEIN_BLACKHOLE_NUMJOBS   I   -   +               N   Y   +
+GLIDEIN_BLACKHOLE_RATE      C   -   +               N   Y   +
```

Comment:
Should these be in the list?
If yes and also to keep them above you should probably assign the value in the condor_config:
```
GLIDEIN_BLACKHOLE_NUMJOBS = $glidein_blackhole_numjobs
GLIDEIN_BLACKHOLE_RATE = $glidein_blackhole_rate
```